src/test/java/org/jsoup/safety/CleanerDeprecatedTest.java

```diff
@@ -0,0 +1,331 @@
+package org.jsoup.safety;
```

Comment:
Per other test comment, this should not be an entire redundant copy. Just the deprecated names.